OsmAnd/res/values/strings.xml

```diff
@@ -51,17 +51,17 @@
     <string name="icon_group_special">Special</string>
     <string name="icon_group_amenity">Amenity</string>
     <string name="app_mode_gap">Gap</string>
-    <string name="ltr_or_rtl_combine_via_dash">%1$s — %2$s</string>
+    <string name="ltr_or_rtl_combine_via_dash">%1$s—%2$s</string>
     <string name="shared_string_local_maps">Local maps</string>
-    <string name="message_graph_will_be_available_after_recalculation">Wait for the route recalculation.\nGraph will be available after recalculation.</string>
-    <string name="message_need_calculate_route_before_show_graph">%1$s data available only on the roads, you need to calculate a route using “Route between points” to get it.</string>
+    <string name="message_graph_will_be_available_after_recalculation">Wait for route recalculation.\nA graph will be available afterwards.</string>
+    <string name="message_need_calculate_route_before_show_graph">%1$s data available only on the roads, calculate a route using \"Route between points\" to get it.</string>
     <string name="shared_string_graph">Graph</string>
     <string name="use_two_phase_routing">Use 2-phase A* routing algorithm</string>
-    <string name="file_already_imported">File is already imported in OsmAnd</string>
-    <string name="osm_edit_logout_success">Logout successful</string>
+    <string name="file_already_imported">File already imported in OsmAnd</string>
+    <string name="osm_edit_logout_success">Logged out</string>
     <string name="clear_osm_token">Clear OpenStreetMap OAuth token</string>
     <string name="perform_oauth_authorization">Log in via OAuth</string>
-    <string name="perform_oauth_authorization_description">Perform an OAuth Login to use osmedit features</string>
+    <string name="perform_oauth_authorization_description">Perform an OAuth login to use osmedit features</string>
```

Comment:
Couldn't find out whether "osmedit" is the correct spelling. "OSMedit" or "OSMEdit" seems more likely.